packages/core/bootstrap/test/unit/validator.test.ts

```diff
@@ -548,6 +620,8 @@ describe('Validator', () => {
       expect(base).toBe('btc')
     })
 
+    /* The following tests have been removed since symbol to symbol and symbol to id overriders
```

Comment:
Something like this you should fully remove the code for and then you can leave this comment on the diff here.

Reply:
I think there may be other adapters still relying on this old code, hence why it has not been removed.  This code will be removed once the overrider is implemented for the remaining adapters.